Ym2149Synth/SynthPatchStorage.h

```diff
@@ -20,6 +20,18 @@
  *
  */
 
+/*
+ * Check for Teensy LC
+ *
+ * LC has only 128 bytes of flash-emulated EEPROM
+ * so we lower the patch amount to 11 instead of 16
+ */
+#if defined (__MKL26Z64__)
+#define MAX_PATCHES 11
+#else
+#define MAX_PATCHES 16
+#endif
+
```

Comment:
Ahh, I see. My mistake there haha.
That's a good idea, should I add that?

And no problem, happy to help - a YM2149 Synth is something I've wanted for a long time.
Thanks to you both for making it easily available.